src/app/settings/index.component.ts

```diff
@@ -289,8 +290,17 @@ export class IndexComponent implements OnInit {
     this.#navigationService.saveSidebar(this.sidebar);
     this.keys = this.#sortKeys(this.#storageService.restoreKeys());
   }
-
+  
+  /**
+   * Remove an item from the sidebar item array with his index number.
+   * Except when equal to "settings", settings item must be available for an easier users navigation.
+   * @param {number} index Index of selected sidebar item
+   * @returns {void}
+  */
   onRemoveSidebarItem(index: number): void {
+    if(this.sidebar[index] === 'settings') {
+      return;
+    } 
```

Comment:
Wouldn't it be better if the settings remove options was totally removed from the GUI ? This solution works but the user can still see it as a bug.
Plus, we can still change the settings button with another one, which is like removing it. Maybe you can try to take a look at `navigation.component` and `navigation.service`